x-pack/docs/en/ml/customurl.asciidoc

```diff
@@ -1,22 +1,48 @@
+[role="xpack"]
 [[ml-configuring-url]]
-=== Adding Custom URLs To Machine Learning Results
+=== Adding custom URLs to machine learning results
 
 When you create an advanced job or edit any job in {kib}, you can optionally
-attach one or more custom URLs. You can also specify these custom settings when
-you create or update jobs by using the {ml} APIs.
+attach one or more custom URLs. 
 
-The custom URLs provide links from the anomalies table in the Anomaly Explorer
-or Single Metric Viewer window in {kib} to custom dashboards or external
+The custom URLs provide links from the anomalies table in the *Anomaly Explorer*
+or *Single Metric Viewer* window in {kib} to custom dashboards or external
```

Comment:
Is it worth adding the Discover page to this sentence? e.g. `... links from the anomalies table in the Anomaly Explorer or Single Metric Viewer to Kibana dashboards, the Discover page or external websites` ?